src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -5,7 +5,7 @@ import styled from "styled-components"
 import { FullArtworkGrid, Tab, Tabs } from "Styleguide/Components"
 import { Col, Row } from "Styleguide/Elements/Grid"
 import { ArtistInfoQueryRenderer as ArtistInfo } from "./Components/ArtistInfo"
-import { ArtworkDetailsFragmentContainer as ArtworkDetails } from "./Components/ArtworkDetails"
+import { ArtworkDetailsQueryRenderer as ArtworkDetails } from "./Components/ArtworkDetails"
```

Comment:
Is this only a query renderer while building out the artwork app or is it intended to stay one?

Reply:
This is new, below the fold react stuff that we're stitching into the old artwork page. The new page will follow a top-down fragment approach.

Reply:
I think it's still to be determined with our work on responsive artwork page @alloy. For now it's actually used in current non reaction Artwork page through sticking so dropped it here to be looking at what production is looking at and for parity with ArtistInfo on this page.

But eventually we might need only one query renderer for the whole artwork page / or not...

Reply:
Gotcha, thanks 👍